AFQ/utils/volume.py

```diff
@@ -42,11 +43,16 @@ def transform_inverse_roi(roi, mapping, bundle_name="ROI"):
     if isinstance(roi, nib.Nifti1Image):
         roi = roi.get_fdata()
 
-    roi = binary_dilation(roi)
-    roi = mapping.transform_inverse(roi, interpolation='linear')
-    roi = patch_up_roi(roi > 0, bundle_name=bundle_name).astype(int)
+    _roi = mapping.transform_inverse(roi, interpolation='linear')
+    _roi = patch_up_roi(_roi > 0, bundle_name=bundle_name).astype(int)
 
-    return roi
+    if np.sum(_roi) == 0:
+        logger.warning(f'warning lost ROI {bundle_name}, performing automatic binary dialtion')
```

Comment:
I think that the logger will tell the user that this is a warning:

```suggestion
        logger.warning(f'Lost ROI {bundle_name}, performing automatic binary dilation)
```